moltres.tex

```diff
@@ -0,0 +1,34 @@
+\begin{frame}
+	\frametitle{Moltres}
+		\begin{itemize}
+			\item Moltres is an application code built on the \gls{MOOSE}
+			framework, for the simulation of \glspl{MSR}
+			\item MOOSE is an open source finite element framework that
+			relies on libMesh and PETSc for advanced meshing and PDE
+			solver capabilities
+			\item Moltres can run transient, fully implicit coupled
+			neutronics/thermal-hydraulics simulations of \glspl{MSR}
+			\begin{itemize}
+				\item Multi-group neutron diffusion (arbitrary groups)
+				\item Delayed neutron precursor advection
+				\item Incompressible Navier-Stokes for temperature
+				advection-diffusion
+				\item 2D axisymmetric or 3D modeling
+			\end{itemize}
+		\end{itemize}
+%		\begin{figure}
```

Comment:
Delete the commented lines if they're used in the very next slide.

Reply:
oh thanks I forgot about those